### ISSUE TITLE
Issues with Speech output in Percussion Customise Kit Dialog

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32789]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

There are a number of non standard and confusing speech issues within the customise kit dialog in the Percussion panel.

STEPS:
1. Create a score with a drum kit and navigate to that staff.  Press Tab several times till you hear percussion, tab twice more to layout and then press down arrow to get to customise Kit.  Press enter.
2. press various arrows, note you don’t seem to be able to navigate and no speech output to know what this is.
3.  Press Tab – you hear “Tree view 56” but no idea what this is or what it is for, e.g. is it a combo box, a spin button and what is its function.
4.  Tab forward a few more times, the rest seem to have some labels, until you get to “None button enter” which makes no sense.
5.  Tab again and hear “Stem direction Combo Box Up collapsed down” Again very confusing are they currently set to up or down?
6.  Tab again and hear “Load Button Enter”, this may make sense if I checked more documentation, but is certainly not obvious from this description of the buttons function.
7.  The following few tabs give the Save As, OK, Cancel, etc all of which weirdly say “button enter” after them, button is OK, but enter is unnecessary as any screen reader user knows to press enter to activate a button it is like telling a person to “click me with mouse”!
8.  Now shift tab back through this dialog and notice that at lots of places it restates the title “customise kit dialog shortcut” before announcing the item.
9.  	Shift Tab back till you get to the place where it announces “Tree View”, press down arrow several times and notice that at no point does this tell you something is selected.  Attempt to select with space and then Tab followed by Shift Tab to confirm speech on this item, it still does not say selected.  There are several list boxes in this dialog with no item showing selected states which makes it very hard to know what is selected when you enter them, especially if you have to use arrows to move to initiate speech.
10.  Tab forward to Note head type and notice it says “Combo Box down” after the selection.  I thought this was referring to stem, but now think it is saying you press a down arrow to open it? If the screen reader says collapsed or list, etc the user will automatically try arrows,  these additional instructions are confusing if it is not clear it is an instruction rather than a description, so either omit or say press down arrow to select.  Again tabbing to the next couple the voice also says down after it, a Braille reader might think this has to do with interval direction or stem direction, etc.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7 beta

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by avvvvve** [comment]
At some point, this entire panel will be replaced with new UI, and when we do so it will need to include proper screen reader output.
